packages/xo-web/src/common/render-xo-item.js

```diff
@@ -13,193 +13,288 @@ import { isSrWritable, subscribeRemotes } from './xo'
-        </span>
-      )}
-    </XoItem>
-  ),
+  ({ template }) => {
+    if (template === undefined) {
+      return UNKNOWN_ITEM
+    }
+
+    return (
+      <span>
+        <Icon icon='vm' /> {template.name_label}
+      </span>
+    )
+  },
 ].reduceRight((value, decorator) => decorator(value))
 
-PoolItem.propTypes = XO_ITEM_PROP_TYPES
+VmTemplate.propTypes = {
+  id: PropTypes.string.isRequired,
+}
+
+VmTemplate.defaultProps = {
+  link: false,
+  newTab: false,
+}
 
 // ===================================================================
 
-export const SrResourceSetItem = [
+export const Sr = [
   connectStore(() => {
     const getSr = createGetObject()
+    const getContainer = createGetObject(
+      createSelector(getSr, sr => get(() => sr.$container))
+    )
     return (state, props) => ({
-      // true to bypass permissions as a self user
-      sr: getSr(state, props, true),
+      // FIXME: props.self ugly workaround to get object as a self user
+      sr: getSr(state, props, props.self),
+      container: getContainer(state, props),
     })
   }),
-  ({ sr, ...props }) => (
-    <XoItem item={sr} to={sr !== undefined && `/srs/${sr.id}`} {...props}>
-      {() => (
-        <span>
-          <Icon icon='sr' /> {sr.name_label || sr.id}
-          {isSrWritable(sr) && (
-            <span>{` (${formatSize(sr.size - sr.physical_usage)} free)`}</span>
-          )}
-        </span>
-      )}
-    </XoItem>
-  ),
+  ({ sr, container, link, newTab }) => {
+    if (sr === undefined) {
+      return UNKNOWN_ITEM
+    }
+
+    return (
+      <LinkWrapper link={link} newTab={newTab} to={`/srs/${sr.id}`}>
+        <Icon icon='sr' /> {sr.name_label}
+        {container !== undefined && (
+          <span className='text-muted'> - {container.name_label}</span>
+        )}
+        {isSrWritable(sr) && (
+          <span>{` (${formatSize(sr.size - sr.physical_usage)} free)`}</span>
+        )}
+      </LinkWrapper>
+    )
+  },
 ].reduceRight((value, decorator) => decorator(value))
 
-SrResourceSetItem.propTypes = XO_ITEM_PROP_TYPES
+Sr.propTypes = {
+  id: PropTypes.string.isRequired,
+  link: PropTypes.bool,
+  newTab: PropTypes.bool,
+}
+
+Sr.defaultProps = {
+  link: false,
+  newTab: false,
+}
 
 // ===================================================================
 
-// Host, Network, VM-template.
-const PoolObjectItem = connectStore(() => {
-  const getPool = createGetObject((_, props) => props.object.$pool)
+export const Vdi = [
+  connectStore(() => ({
+    vdi: createGetObject(),
+  })),
+  ({ vdi }) => {
+    if (vdi === undefined) {
+      return UNKNOWN_ITEM
+    }
 
-  return (state, props) => ({
-    pool: getPool(state, props),
-  })
-})(({ object, pool }) => {
-  const icon = OBJECT_TYPE_TO_ICON[object.type]
-  const { id } = object
+    return (
+      <span>
+        <Icon icon='disk' /> {vdi.name_label}
+        {vdi.name_description && <span> ({vdi.name_description})</span>}
```

Comment:
why you didn't display the SR instead description?

Reply:
I kept it as it was. I'm not trying to change the behaviour in this PR.